FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/MessageDetailsPassphraseInRamFlowTest.kt

```diff
@@ -220,6 +222,8 @@ class MessageDetailsPassphraseInRamFlowTest : BaseMessageDetailsFlowTest() {
   }
 
   @Test
+  @FlakyTest
+  @Ignore("failed sometimes on CI")
```

Comment:
Maybe need to make new issue for this `Ignore`?

Reply:
we already have this one https://github.com/FlowCrypt/flowcrypt-android/issues/2038